java-spring/accounts-command-side-backend/src/main/java/net/chrisrichardson/eventstore/javaexamples/banking/backend/commandside/accounts/AccountWorkflow.java

```diff
@@ -22,7 +22,13 @@
 
     String fromAccountId = event.getDetails().getFromAccountId();
 
-    return ctx.update(Account.class, fromAccountId, new DebitAccountCommand(amount, transactionId));
+    return ctx.update(Account.class, fromAccountId, new DebitAccountCommand(amount, transactionId)).handle((x, e) -> {
+              if (e != null) {
+                e.printStackTrace();
```

Comment:
I used this to handle old events with wrong content. Otherwise my eventHandler methods produced runtime exceptions. I'll remove the handle(...) code